src/app/logging/log-table/log-table.component.ts

```diff
@@ -102,6 +102,13 @@ export class LogTableComponent implements OnInit {
     })
   }
 
+  /**
+   * Assigns a Label that will be displayed in the log table for each LogType
+   * 
+   * @param logType that should be converted
+   * 
+   * @returns a Label for each LogType
```

Comment:
zwischen den einzelnen tags muss keine leerzeile sein (nur zwischen den tags und der darüberliegenden description) but anyway.
```suggestion
   * @param logType that should be converted
   * @returns a Label for each LogType
```